app/konnect/centralized-consumer-management.md

```diff
@@ -0,0 +1,78 @@
+   * `$CONSUMER_GROUP`: (Optional) Replace with the name of the consumer groups you want to associate with the realm.
+1. Use the `/realms/{realmId}/consumers` endpoint to create a centrally managed consumer and optionally assign it to a consumer group:
+   ```
+   curl -X POST \
+   https://{region}.api.konghq.com/v1/realms/{realmId}/consumers \
+   -H "Content-Type: application/json" \
+   -H "Authorization: Bearer TOKEN" \
+   -d '{
+         "username": "$CONSUMER_NAME",
+         "consumer_groups": ["$CONSUMER_GROUP"]
+       }'
+   ```
+   Be sure to replace the following with your own values:
+   * `{region}`: Region for your {{site.konnect_short_name}} instance.
+   * `$KONNECT_TOKEN`: Replace with your {{site.konnect_short_name}} personal access token.
+   * `{realmId}`: The ID of the realm you created previously. 
+   * `$CONSUMER_NAME`: Replace with the name of the consumer.
+   * `$CONSUMER_GROUP`: (Optional) Replace with the name of the consumer groups you want to associate with the consumer. Consumer groups set here are additive. This means that if you configure the realm with consumer groups A and B, and then configure the consumer with consumer group C, the authenticated consumer will be assigned to consumer groups A, B, and C.
+1. Consumers require authentication. Configure authentication using the [Key Auth plugin](/hub/kong-inc/key-auth/how-to/).
+
+   `identity_realms` are scoped to the control plane by default (`scope: cp`). The order in which you configure the `identity_realms` dictates the priority in which the data plane attempts to authenticate the provided API keys:
+
+    * **Realm is listed first:** The data plane will first reach out to the realm. If the API key is not found in the realm, the data plane will look for the API key in the control plane config. 
+    * **Control plane scope listed first:** The data plane will initially check the control plane configuration (LMDB) for the API key before looking up the API Key in the realm.
+    * **Realm only:** You can also configure a single `identity_realms` by omitting the `scope: cp` from the example. In this case, the data plane will only attempt to authenticate API keys against the realm. If the API key isn't found, the request will be blocked.
+    * **Control plane only:** You can configure a look up only in the control plane config by only specifying `scope: cp` for `identity_realms`. In this scenario, the data plane will only check the control plane configuration (LMDB) for API key authentication. If the API key isn't found, the request will be blocked.
+
+{:.note}
+> **Note:** If you are using KIC to manage your data plane nodes in {{site.konnect_short_name}}, ensure that you configure the `telemetry_endpoint` in the data plane. You can find the `telemetry_endpoint` in the {{site.konnect_short_name}} UI in [Gateway Manager](https://cloud.konghq.com/gateway-manager/) in the data plane node instructions.
+
```

Comment:
Do we want to add a section around changing the scope of consumers from cp to realm? The user will need to do the following call:

POST https://us.api.konghq.com/identity/pools/%7BpoolID%7D/consumers)/Alice-123/keys 
{
	"type": "legacy" // Migrating Alice's existing keys
	"secret": "Alice-api-key"
}

Reply:
I am adding a new step to create new keys for the consumer, since I think that was missing.